### DISCUSSION TITLE
[Q&A] 自部署流式返回这个在哪里修改，在项目的哪里

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1098] [Self Hosting]
#531 这个我看过了，但是不清楚在哪里修改，希望告知

**Comment by 5478-lfy** [comment]
我这样做是因为，我在接上Azure OpenAI GPT4的api接口之后，可以使用，但是响应速度太慢，所以我想要做一下修改

**Reply by 5478-lfy** [reply]
主要是开始的响应速度太慢了，有时需要半分钟左右甚至更长的时间，这个是无法接受的

**Comment by arvinxx** [comment]
你是用nginx 吗？是的话就nginx 的配置文件里呀

**Reply by 5478-lfy** [reply]
nginx 部署有教程吗？谢谢

**Reply by arvinxx** [reply]
https://github.com/lobehub/lobe-chat/discussions/531
